test/1_stdlib/Inputs/ProcessStressTest/ProcessStressTest.c

```diff
@@ -0,0 +1,48 @@
+#include <CoreFoundation/CoreFoundation.h>
```

Comment:
I can get the test to break early if I refactor to remove the set and instead keep around an atomic reference to the bits of the pointer.  Condition can be failing an atomic compare.